dwave/gate/simulator/simulator.pyx

```diff
@@ -178,6 +185,52 @@ def _measure(op, state, targets):
         op._measured_qubit_indices.append(t)
 
 
+def _fast_sample(
```

Comment:
🐎💨

Reply:
I can only see this image as a horse farting and I am here for it.

Reply:
My thoughts exactly.